pkg/model/api/config/config_test.go

```diff
@@ -68,6 +68,10 @@ func TestJSONMarshalConfig(t *testing.T) {
     "disable": false,
     "explorer": {
       "enable": false
+    },
+    "time_range": {
+      "disable_custom": false,
+      "options": null
```

Comment:
Should be empty array, no?

Reply:
the zero value of a slice is nil in Golang. I think this is the reason why.

But anyway it doesn't happen in real life as the Verify will always override with default value, options being nil or empty

Reply:
With omitempty it's even better, not there :)